### ISSUE TITLE
Default menu on input control does not work

### ISSUE THREAD

**Issue by asheshv** [WebCabin/wcDocker#83]
Default menu of an input control does not work when enabled the docker menu.

**Comment by Lochemage** [comment]
@asheshv 

That is correct, what I meant by **"default"** is not the true default menu options you would see built into the browser, but rather the default menu options you normally see provided by `wcDocker`. For example, **"Remove Panel"** or **"Add Panel"**.

I currently do not know how to re-implement browser commands like copy, paste, etc. I'm not even sure if it is possible.

**Comment by Lochemage** [comment]
@asheshv 

If it is absolutely necessary that you have the browsers default menu, you can try using `wcDocker.bypassMenu()`. The definition can be found here: http://docker.api.webcabin.org/module-wcDocker.html#toc12
